src/components/formFields/UploadField.tsx

```diff
@@ -0,0 +1,64 @@
+import React from "react";
+import { FormLabel, FormControl, Image, Input, FormErrorMessage } from "@chakra-ui/react";
+import { getPresignedPostForUpload } from "@aws/s3";
+
+type Props = {
+    value: string;
+    setValue: (text: string) => void;
+    name: string;
+    required?: boolean;
+    edit?: boolean;
+};
+
+export const UploadField: React.FC<Props> = ({
+    value,
+    setValue,
+    name,
+    required = true,
+    edit = true,
+    ...props
+}): JSX.Element => {
+    const upload = (value) => {
+        console.log(value.target.files); //returns FILELIST array (will only be the first element)
+        console.log(process.env.S3_PUBLIC_IMAGES_BUCKET);
+        const post = getPresignedPostForUpload("sdc-public-images", "Test/image.jng");
+
+        //setValue(fileUrl)
+    };
+
+    return (
+        <FormControl
+            isRequired={required && edit}
+            isInvalid={!value && required && edit}
+            {...props}
+        >
+            <FormLabel>{name}</FormLabel>
+            <Image
+                boxSize="200px" //TODO: Figure out how to set width/length seperately
+                objectFit="cover"
+                src={value}
+                alt="Segun Adebayo"
+            />
```

Comment:
Here's a solution!
We can pass a `isLarge` prop into this component and depending on weather or not is large we can do something like
```suggestion
            <AspectRatio ratio={2} width="350px">
                <Image src={value} alt="Segun Adebayo" />
            </AspectRatio>
```
if it is large.

![image](https://user-images.githubusercontent.com/44826218/143988978-2c35edb9-227b-4815-a6c2-6a09886721c5.png)

Reply:
I'll explore this